src/Compilers/Shared/CoreClrShim.cs

```diff
@@ -15,5 +16,15 @@ internal static class AssemblyLoadContext
             internal static readonly Type Type = ReflectionUtilities.TryGetType(
                "System.Runtime.Loader.AssemblyLoadContext, System.Runtime.Loader, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a");
         }
+
+        internal static class AppContext
+        {
+            internal static readonly Type Type = ReflectionUtilities.TryGetType(
+                "System.AppContext, System.AppContext, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a");
```

Comment:
The API we need here is from netstandard 1.6, which is in contract assembly version 4.1.0.0; netstandard 1.3 assembly version is 4.0.0.0 and doesn't have this API.
The package we reference in project.json (v4.3.0) contains both contract assemblies.